### PR TITLE
Fix image zoom boundries

### PR DESCRIPTION
**Pull request by askeland**
Better late than never 🙈

### CODE REVIEW

File: Source/ViewableController.swift [hunk 1]
```diff
@@ -35,6 +35,10 @@ class ViewableController: UIViewController {
         scrollView.minimumZoomScale = 1.0
         scrollView.maximumZoomScale = self.maxZoomScale()
         scrollView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        
+        if #available(iOS 11.0, tvOS 11.0, *) {
+            scrollView.contentInsetAdjustmentBehavior = .never
```
Comment (elland) [inline Source/ViewableController.swift:40]: 😭

**Comment by elland** [comment]
💪🏽

**Comment by elland** [comment]
So this fixes some of the zoom issues. But behaviour between first loaded image and subsequent images after you swipe left/right still persists.

Double tapping anywhere on the screen for the first image zooms it left-aligned, and trying to pan around dismisses. Works as expected elsewhere. Any idea?

**Comment by askeland** [comment]
Ah, you are right. I remember that problem. I think I managed to add some more of the missing code now...

**Comment by elland** [comment]
Sorry, accidentally merged this. 🤦🏽‍♂️

In any case, that did not have the desired effect. Now I can pan to the next image, instead of around the currently zoomed image, to focus on different sections… 😅

**Comment by askeland** [comment]
Hmm, it is working as expected here on my simulator. I can zoom in and pan around. When you hit the boundary, you can swipe to the next. Maybe I misunderstand what you mean?

**Comment by elland** [comment]
On the very first image or after swiping to a different image? Tapping an image in the collection view, then double tapping anywhere zooms in on the leftmost side and I can't pan around. 🤔(iOS11)

**Comment by askeland** [comment]
Ah, you are right. I was testing with the local photos. Was able to reproduce now with the remote ones. Not sure why that happen though. Need to look into it later, but feel free to explore in the meantime 😬

**Comment by elland** [comment]
Yeah! No luck on my end so far…

**Comment by askeland** [comment]
My first guess was that the contentview of the UIScrollView is the wrong size, but as far as I can see, it looks correct.

**Comment by marijnschilling** [comment]
screenshot 2017-10-18 17 28 17
screenshot 2017-10-18 17 28 02

But now it always zooms into the left size always. To a x=0 position...

**Pull request merged**